subprojects/core/src/main/java/org/gradle/process/internal/worker/WorkerProcessBuilder.java

```diff
@@ -59,6 +59,18 @@ public interface WorkerProcessBuilder extends WorkerProcessSettings {
 
     void enableJvmMemoryInfoPublishing(boolean shouldPublish);
 
+    // TODO: Remove this option for Gradle 8.0 and do not use the args at all.
+    // We are leaving these args enabled for most worker types in order to reduce breaking changes for those using
```

Comment:
Wouldn't we want to see these notes about why this method is present in the normal javadoc?  Why separate line comments?